net/sshuttle/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
```

Comment:
```suggestion
PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
PKG_LICENSE:=LGPL-2.1-only
PKG_LICENSE_FILES:=LICENSE

```

Reply:
Thanks! I added the line for the license file